// !API_VERSION: 1.2

val a = <!INT_LITERAL_OUT_OF_RANGE, UNSIGNED_LITERAL_WITHOUT_DECLARATIONS_ON_CLASSPATH!>0u<!>
val b = <!INT_LITERAL_OUT_OF_RANGE, UNSIGNED_LITERAL_WITHOUT_DECLARATIONS_ON_CLASSPATH!>1uL<!>

fun foo(<!UNUSED_PARAMETER!>u<!>: <!UNRESOLVED_REFERENCE!>UInt<!>) {}